test/e2e/multi-select.test.js

```diff
@@ -9,8 +9,146 @@ describe('vl-multi-select', async () => {
+        const multiselect = await vlMultiSelectPage.getDisabledMultiselect();
+        await assert.eventually.isTrue(multiselect.isDisabled());
+    });
+
+    it('Het aantal resultaten van een zoekopdracht kan beperkt worden', async () => {
+        const multiselect = await vlMultiSelectPage.getMultiselectMetSpecifiekAantalResultaten();
+        await multiselect.searchByPartialText('straat');
+        await assert.eventually.equal(multiselect.getNumberOfSearchResults(), 5);
+    });
+
+    it('Het aantal resultaten van een zoekopdracht is standaard niet beperkt', async () => {
+        const multiselect = await vlMultiSelectPage.getMultiselectMetOnbeperkteResultaten();
+        await multiselect.searchByPartialText('straat');
+        await assert.eventually.isAbove(multiselect.getNumberOfSearchResults(), 5);
+    });
 
-    it("Dummy test om browsers te sluiten", () => {
-    	assert.isTrue(true);
+    it('Als gebruiker kan ik luisteren naar een change event en onChange zal er een extra attribuut gezet worden', async () => {
+        const multiselect = await vlMultiSelectPage.getChangeEventMultiselect();
+        await assert.eventually.isFalse(multiselect.hasAttribute('changed'));
+        await multiselect.selectByValue('Germany');
+        await assert.eventually.isTrue(multiselect.hasAttribute('changed'));
     });
+
+    it('Als gebruiker kan ik de multiselect via enable/disable-methode in/uit-schakelen', async () => {
+        const multiselect = await vlMultiSelectPage.getEnableDisableMethodeMultiselect();
+        await assert.eventually.isFalse(multiselect.isDisabled());
+        await vlMultiSelectPage.disable();
+        await assert.eventually.isTrue(multiselect.isDisabled());
+        await vlMultiSelectPage.enable();
+        await assert.eventually.isFalse(multiselect.isDisabled());
+    });
+
+    it('Als gebruiker kan ik programmatorisch (een) waarde(s) toevoegen/verwijderen in de multiselect', async () => {
+        const multiselect = await vlMultiSelectPage.getSetGetMethodeMultiselect();
+        await assert.eventually.isEmpty(multiselect.getSelectedItems());
+        await vlMultiSelectPage.kiesDuitsland();
+        const de = await multiselect.getSelectedItems();
+        assert.isTrue(de.some(i => i.text === 'Duitsland'));
+        await vlMultiSelectPage.verwijderSelectie();
+        await assert.eventually.isEmpty(multiselect.getSelectedItems());
+        await vlMultiSelectPage.kiesBelgieEnNederland();
+        const nlBe = await multiselect.getSelectedItems()
+        assert.isTrue(nlBe.some(i => i.text === 'België'));
+        assert.isTrue(nlBe.some(i => i.text === 'Nederland'));
+    });
+
+    // Testen mogen terug uit commentaar gehaald worden wanneer er een stabiele versie van de datepicker is
+    // it('Als een multiselect boven een datepicker gerenderd wordt, kunnen zowel de multiselect als de datepicker correct gebruikt worden', async () => {
```

Comment:
En de gebruikte branch van datepicker is niet voldoende om deze uit commentaar te halen?

Reply:
subticket gemaakt onder de E2E testen om dit nog te bekijken @Pieter-Beckers